### DISCUSSION TITLE
What does the percentage number mean?

### DISCUSSION THREAD

**Discussion by Tealk** [Xyphyn/photon#181] [Q&A] [answered]
![image](https://github.com/Xyphyn/photon/assets/12276250/93317109-3202-4aab-a35b-a0d3084e73c2)

**Comment by Xyphyn** [comment, marked as answer]
It's the upvote ratio. It's how many of the total votes were upvotes, and can typically represent most people's opinion on a submission.

**Reply by Tealk** [reply]
Could you show a note as alt text/mouseover? Then it would be clearer.
